apps/condo/domains/notification/schema/BlackList.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Generated by `createschema notification.BlackList 'organization?:Relationship:Organization:SET_NULL; user?:Relationship:User:CASCADE; phone?:Text; email?:Text;'`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/notification/access/BlackList')
+const { MESSAGE_TYPES } = require('@condo/domains/notification/constants/constants')
+
+const BlackList = new GQLListSchema('BlackList', {
```

Comment:
It’s better to add some xtra cord here. It’s context less. When you see the name blackList it’s hard to understand that it’s about messages. My be something like “send message black list” or “message black list”

Reply:
renamed to MessageBlackList